src/All-Time-Hydration.js

```diff
@@ -0,0 +1,13 @@
+class AllTimeHydration {
+  constructor() {
+
+  }
+  returnAllTimeHydration(hydrationData, id){
+    return hydrationData.filter(dailyHydration => dailyHydration.userID === id);
```

Comment:
This will filter through the 5000 hydration objects and return those only belonging to the specific ID. (should be 100 in the test data).

Mike and I spitballed a bit about how we want our Data Flow to work, and decided that we don't need to ever store this data, but rather return it like this method does.

This method will be the top of our funnel for the flow, as we believe we will never need a larger scope of data than the 100 (or all) hydration objects returned the all-time data.
We will use this return value to calculate our daily, and weekly stats. (maybe we should store this instead of having to call it every time now that I think about it.)

Reply:
Good work.  I like the way that you set up the testing data files in different describe blocks.